src/wp-includes/pomo/streams.php

```diff
@@ -11,8 +11,8 @@
 if ( ! class_exists( 'POMO_Reader', false ) ) :
 	class POMO_Reader {
 
-		var $endian = 'little';
-		var $_post  = '';
+		public $endian = 'little';
+		public $_post  = '';
```

Comment:
Do you think these PHP 4 poor mans protected properties should be marked `protected`? I can see this could be a technical BC break, but we wouldn't be able to mark them `protected` at a later time, as it is now explicitly declared `public`?

Reply:
It would be great if we could, however, it would be a BC-break (not just a technical one) as this code has been in WP since forever and `var` translates to `public`, so they have been in the codebase as `public` since forever.
As we don't know whether people have taken advantage of that in one way or another, changing it to `protected` now, would break such code.

Reply:
I wish it was different too...

Reply:
I see and agree. Declaring the properties with public in 5.6 would be a huge step forward even if we were to mark all current properties `public` to save from BC breaks. Thanks a lot for opening this 🙏